ShaiRandom/Generators/IEnhancedRandomExtensions.cs

```diff
@@ -294,7 +294,7 @@ public static T RandomElement<T>(this IEnhancedRandom rng, ReadOnlySpan<T> items
         /// <typeparam name="T">The type of items in the span.</typeparam>
         /// <param name="items">Must be non-empty.</param>
         /// <param name="selector">Function that should return true _only_ for a valid item to select.</param>
-        /// <returns>A randomly-chosen item from the span for which <see cref="selector"/> returns true.</returns>
+        /// <returns>A randomly-chosen item from the span for which <paramref name="selector"/> returns true.</returns>
```

Comment:
I see these are all fixed now. I should *not* be going through this by each commit... I had marked each place that used see instead of paramref, but you already got all of those.